### PR TITLE
cmd/tailscale/cli: fix inconsistency between serve text and example command

### PR DESCRIPTION
**Pull request by mihaip**
Use the same local port number in both.

### CODE REVIEW

File: cmd/tailscale/cli/serve.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ EXAMPLES
     $ tailscale serve https:8080 / text:"Hello, world!"
 
   - To forward raw TCP packets to a local TCP server on port 5432:
-    $ tailscale serve tcp:2222 tcp://localhost:22
+    $ tailscale serve tcp:2222 tcp://localhost:5432
```
Comment (mihaip) [inline cmd/tailscale/cli/serve.go:70]: Not sure if the 2222 should also be called out in the text as the incoming port.

Comment (bradfitz) [reply cmd/tailscale/cli/serve.go:70]: The line before is also slightly misleadingly wrong. It's not forwarding "raw TCP packets"; it's reconstruction a new TCP stream and proxying bytes between them.

"To forward incoming TCP connections to a local TCP server on ..."

Comment (mihaip) [reply cmd/tailscale/cli/serve.go:70]: Should the `tls-terminated-tcp` option below get a similar change?

Comment (bradfitz) [reply cmd/tailscale/cli/serve.go:70]: yup

Comment (mihaip) [reply cmd/tailscale/cli/serve.go:70]: Done.

File: cmd/tailscale/cli/serve.go [hunk 2]
```diff
@@ -66,10 +66,10 @@ EXAMPLES
   - To serve simple static text:
     $ tailscale serve https:8080 / text:"Hello, world!"
 
-  - To forward raw TCP packets to a local TCP server on port 5432:
-    $ tailscale serve tcp:2222 tcp://localhost:22
+  - To forward incoming TCP connections on port 2222 to a local TCP server on port 5432:
```
Comment (bradfitz) [inline cmd/tailscale/cli/serve.go:69]: this line's now 88 chars wide. From lines above, it looks like we're trying to be 80 char friendly. (some people configure their tiling window managers to have NxM terminals where each is 80 cols wide)

File: cmd/tailscale/cli/serve.go [hunk 3]
```diff
@@ -66,10 +66,10 @@ EXAMPLES
...
+    $ tailscale serve tcp:2222 tcp://localhost:5432
 
-  - To forward raw, TLS-terminated TCP packets to a local TCP server on port 80:
+  - To forward incoming TLS-terminated TCP connections to a local TCP server on port 80:
```
Comment (bradfitz) [inline cmd/tailscale/cli/serve.go:72]: I think the word order is wrong now for what this mode does.

Maybe: "To accept TCP TLS connections, terminate their TLS inside tailscaled, and then proxy to a local plaintext server on port 80, wrapping the plaintext response in TLS to the client:"

Or something. @shayne can help wordsmith.

Comment (shayne) [reply cmd/tailscale/cli/serve.go:72]:
```suggestion
  - Accept TCP TLS connections (terminated within tailscaled) proxied to a local plaintext server on port 80. 
  ```

**Comment by shayne** [comment]
Regarding the `$ tailscale serve tcp:2222 tcp://localhost:22`, this was a use case/customer question that popped up a couple of times. It allows you to rebind OpenSSH to port 2222 on your tailnet, allowing access to the real SSH server if you're running Tailscale SSH.

Access isn't allowed via Funnel (non-TLS and the port is not allowed), but for serving only on your tailnet, you can proxy non-TLS-wrapped TCP connections on any valid port. Before the recent CLI updates, `serve` restricted `--serve-port` to the three we allowed on Funnel (`443,8443,10000`).

**Review by shayne — APPROVED** [review]
Thanks for the updates! I added a comment and suggestion—otherwise, LGTM.

**Comment by shayne** [comment]
Typo in commit message:
> fix inconsistency **between** serve text and ...

File: cmd/tailscale/cli/serve.go (hunk 3 above)
Comment (mihaip) [reply cmd/tailscale/cli/serve.go]: Incorporated Shayne's suggestion.

**Comment by mihaip** [comment]
> Regarding the `$ tailscale serve tcp:2222 tcp://localhost:22`, this was a use case/customer question that popped up a couple of times. It allows you to rebind OpenSSH to port 2222 on your tailnet, allowing access to the real SSH server if you're running Tailscale SSH.

Mentioned this more explicitly, I think it's a good use case that we should document.

**Pull request merged**